core/build.gradle.kts

```diff
@@ -88,6 +89,12 @@ kotlin {
             languageSettings {
                 optIn("kotlinx.cinterop.ExperimentalForeignApi")
             }
+
+            val commonMain by sourceSets.getting {
+                dependencies {
+                    implementation(libs.configuration.annotations)
```

Comment:
For my understanding, why can't this dependency be included in the `commonMain.dependencies`?

Reply:
This was the recommendation from the docs on how to include it so I followed that. Do you think there's a better way to do this?

Reply:
In the same file a couple of lines down `commonMain.dependencies` there are dependencies declared there. Can't we also include this there?

Reply:
Done, thanks!